src/main/kotlin/io/snyk/plugin/SnykBulkFileListener.kt

```diff
@@ -71,14 +81,49 @@ class SnykBulkFileListener() : BulkFileListener {
                 }
 
             if (filesToRemoveFromCache.isNotEmpty())
-                project.service<SnykTaskQueueService>().scheduleRunnable("SnykCode is updating caches...") {
+                project.service<SnykTaskQueueService>().scheduleRunnable("Snyk Code is updating caches...") {
                     if (filesToRemoveFromCache.size > 10) {
                         // bulk files change event (like `git checkout`) - better to drop cache and perform full rescan later
                         AnalysisData.instance.removeProjectFromCaches(project)
                     } else {
                         AnalysisData.instance.removeFilesFromCache(filesToRemoveFromCache)
                     }
                 }
+
+            // clean .dcignore caches if needed
```

Comment:
Maybe it would be better to "hide" all logic related to dc ignore functionality in a separate class? Something like DcIgnoreService?